### PR TITLE
Adds 15 Pride flags to countrybox selection 

### PR DESCRIPTION
**Pull request by chesse20**
~~I've never used maven (and "mvn install -DskipTests" doesn't seem to do anything in visual studio code) and i haven't touched java in like 3 years but after looking over the code this might add~~ this adds 15 pride flags to select instead of countries, for those who care more about their pronouns, identity, and/or support for their lgbtq+ friends over displaying their country of origin. 

https://user-images.githubusercontent.com/9161779/235375607-1339576b-fedc-42f4-b217-fddef35b5f21.mp4

### CODE REVIEW

**Comment by chesse20** [comment]
image
i got step on this step in the visual studio code set-up so i can't test the code yet

**Comment by chesse20** [comment]
> > image
> > i got step on this step in the visual studio code set-up so i can't test the code yet
> 
> I haven't tried VSC since I only use IntelliJ.
> 
> Have you tried the instructions [here](https://github.com/magefree/mage/wiki/Setting-up-your-Development-Environment#visual-studio-code-vsc)?

yeah i followed the instructions a bit better and got to around step 9 but got a bunch of errors and couldn't figure out how to get the client to run. ill try it again tommorow

**Comment by chesse20** [comment]
image
idk how to run the client from here

**Comment by Alex-Vasile** [comment]
> image
> idk how to run the client from here



> image
> idk how to run the client from here

I would venture a guess and say that you have to hover over "mage-client" in the "JAVA PROJECT" dropdown there and press the play button (after you've built it with maven).

**Comment by Alex-Vasile** [comment]
> image
> idk how to run the client from here



> image
> idk how to run the client from here

I would venture a guess and say that you have to hover over "mage-client" in the "JAVA PROJECT" dropdown there and press the play button (after you've built it with maven).

**Comment by peter-thoenen** [comment]
Can we vote no to not "woken" Xmage, if I wanted that I would go give Hasbro my money.  Last I checked we already have a pride flag, it's called France.  We don't need more because it opens up the flood doors to every one elses pet flags and causes. Do we add political party flags? universities? etc etc.

**Comment by chesse20** [comment]
im gonna try this with intelej, been busy with finals so i haven't had much energy but im trying to get this done before pride month

File: Mage.Client/src/main/java/mage/client/util/gui/countryBox/CountryComboBox.java [hunk 1]
```diff
@@ -185,6 +198,7 @@ public class CountryComboBox extends JComboBox {
         {"Palau", "pw"},
         {"Palestine, State of", "ps"},
         {"Panama", "pa"},
+        {"Pansexual","pan"}
```
Comment (the-red-lily) [inline Mage.Client/src/main/java/mage/client/util/gui/countryBox/CountryComboBox.java:201]: Missing ,

**Review by the-red-lily — CHANGES_REQUESTED** [review]

**Comment by chesse20** [comment]
i fixed some issues with the file and managed to compile the master branch, but something about this code causes the server files to not compile. i got no clue whats wrong with

**Review by xenohedron — APPROVED** [review]
I tested this and found no issues. Hope it can be merged soon.

**Comment by Grath** [comment]
Can someone compile it and test with a server? I'm seeing the pride flags in the client in the Connect window, but I don't see them rendering properly when connected to the server (and I don't know what's broken there, whether it's my end or an issue with this pull request.)

**Review by xenohedron — COMMENTED** [review]
@chesse20 @Grath
I tested more thoroughly and also found that the flag images are not rendering correctly once connected to the server. I guess more investigation is needed there.

**Comment by chesse20** [comment]
yeah i can't find any flag code related to the server so im confused what i could be missing

**Comment by alexander-novo** [comment]
Works fine for me

image

image

I merged master in to the branch before building.

**Comment by xenohedron** [comment]
Great! I just tested it again and it worked fine for me this time. No idea what was going on before.

**Pull request merged**